Remotion/Validation/Core/Attributes/Validation/NotEmptyValidationAttribute.cs

```diff
@@ -38,20 +65,34 @@ protected override IEnumerable<IPropertyValidator> GetValidators (IPropertyInfor
       ArgumentUtility.CheckNotNull("property", property);
       ArgumentUtility.CheckNotNull("validationMessageFactory", validationMessageFactory);
 
-      NotEmptyValidator validator;
+      Func<ValidationMessage, IPropertyValidator> validatorFactory;
+      if(typeof(string) == property.PropertyType)
```

Comment:
```suggestion
      if(property.PropertyType == typeof(string))
```
The compared-to data is on the right-side whenever syntactically possible to correspond with natural language / mental flow.